Dockerfile

```diff
@@ -1,8 +0,0 @@
-FROM quarktheawesome/devkitpro-arm-ppc:r56_r39
-
-RUN apt-get update && apt-get install -y --no-install-recommends \
-  xxd \
-  && rm -rf /var/lib/apt/lists/*
-
-WORKDIR /app
-CMD make -j$(nproc)
```

Comment:
Should keep the Dockerfile too, it helps a lot for people who don't have the perfect setup

Reply:
Imma need some help on that